modules/auth_saml/spec/features/administration/saml_crud_spec.rb

```diff
@@ -47,7 +47,7 @@
       fill_in "Name", with: "My provider"
       click_link_or_button "Continue"
 
-      expect(page).to have_css("h1", text: "My provider")
+      expect(page).to have_test_selector("saml-provider-page-header-title", text: "My provider")
```

Comment:
You can use a more general [Capybara Accessible Selector](https://github.com/citizensadvice/capybara_accessible_selectors) here instead:

```suggestion
      expect(page).to have_heading "My provider"
```